lambdas/src/handlers/access-token-handler.ts

```diff
@@ -33,10 +33,12 @@ export class AccessTokenLambda implements LambdaInterface {
         try {
             await initPromise;
 
+            logger.info("Access Token Lambda triggered with event", JSON.stringify(event));
```

Comment:
What is the full set of properties in the event, do we need them all?

Also, I can't remember - does the `AWS logger` accept an object for the 2nd property, or does it need to be `Stringified`? It'd be more performant if it could remain as JSON.

Reply:
It only accepts strings unfortunately, I'll strip out the bits of the event it actually uses and log those instead of the whole object